investd/sources/base.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 from typing import Generator, Iterable
 
-from ..config import SOURCE_BASE_PATH
+from ..config import INVESTD_SOURCES
```

Comment:
how about making absolute imports from "investd" as in other files?

Reply:
Changed everything to absolute